docs/scos/dev/technical-enhancement-integration-guides/configuring-dynamic-yves-zed-tokens.md

```diff
@@ -37,6 +37,7 @@ After running `docker/sdk bootstrap`, you can find all the tokens in environment
 
 | Parameter | Parameter description | Env variable | Description of env variable value  |
 | --- | --- | --- | --- |
+| `HttpConstants::URI_SIGNER_SECRET_KEY` | Encrypts URI signs. | `SPRYKER_URI_SIGNER_SECRET_KEY` | Consists of 80 characters in lower and upper case Latin letters and digits. |
```

Comment:
I don't think that we need to stick to the specific amount of characters. 
General rule is enough from my side.